character-sfx.js

```diff
@@ -82,7 +82,7 @@ class CharacterSfx {
     this.lastEmote = null;
 
     const wearupdate = e => {
-      sounds.playSoundName(e.wear ? 'itemEquip' : 'itemUnequip');
+      sounds.playSoundName(e.wear ? 'itemEquip' : 'itemUnequip', {voicer: this.player});

```

Comment:
I don't think `voicer` is a good name for this, since it is not voice. That is confusing since the player also has a voicer and it is completely unrelated to this.

Can we call it `soundEmitter` or something?

Reply:
change the voicer to soundEmitter

Reply:
[change the voicer to soundEmitter](https://github.com/webaverse/app/pull/3056/commits/56ecf19ab811c0961ba52953a1399f3a3e92bf3f)